### PR TITLE
Move whitepapers to seperate page

### PR DESCRIPTION
**Pull request by angusfretwell**
TODO:

- [ ] Russian translation
- [ ] Chinese translation
- [ ] Update routing rules (for whitepapers.html redirect)

### CODE REVIEW

File: src/locales/zh.js [hunk 1]
```diff
@@ -120,7 +120,11 @@ export default {
       download: '下载',
       torrent: '种子',
     },
-    whitepapers: {
+  },
+  whitepapers: {
+    title: 'TODO: Whitepapers',
+    description: 'TODO: Download Skycoin whitepapers.',
```
Comment (angusfretwell) [inline src/locales/zh.js]: Translation needed

File: src/locales/ru.js [hunk 2]
```diff
@@ -120,7 +120,11 @@ export default {
       download: 'Скачать',
       torrent: 'Torrent',
     },
-    whitepapers: {
+  },
+  whitepapers: {
+    title: 'TODO: Whitepapers',
+    description: 'TODO: Download Skycoin whitepapers.',
```
Comment (angusfretwell) [inline src/locales/ru.js]: Translation needed

Comment (angusfretwell) [inline src/locales/ru.js]: @gogo-tattoo Could you add a translation for this?

**Comment by gz-c** [comment]
what happened to this

**Comment by gz-c** [comment]
outdated

**Pull request closed without merging**